bundles/CoreBundle/EventListener/Frontend/StaticPageGeneratorListener.php

```diff
@@ -93,7 +94,19 @@ public function onKernelRequest(RequestEvent $event)
         $storage = Storage::get('document_static');
 
         try {
-            $filename = urldecode($request->getPathInfo()) . '.html';
+            $path = '';
+            $filename = urldecode($request->getPathInfo());
+
+            if (Site::isSiteRequest()) {
+                $path = Site::getCurrentSite()->getRootPath();
```

Comment:
I would move this in else part to save an extra call in case of `$request->getPathInfo() === '/'`

Reply:
@dvesh3 good catch. done